### PR TITLE
NTP: Update dashboard interface

### PR DESCRIPTION
**Pull request by cezaraugusto**
Close brave/brave-browser#9455

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [x] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:

See screenshot attached.

ntp_dashboard

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by simonhong** [comment]
Previously, whole `Background Images` part was hidden when Super Referrals are activated like below. How it's handled in this new design?
image

**Comment by rebron** [comment]
cc: @karenkliu How do we want to handle hiding `Background Image` section when super referrals are enabled? Hide background image on the list and bump everything up?

**Comment by karenkliu** [comment]
@rebron Yes, correct.

**Comment by bsclifton** [comment]
Really nice improvement- styles look a little weird on Windows (not sure why it's not using translations either)
image

**Comment by karenkliu** [comment]
@bsclifton Here's some additional design QA feedback: 

design QA 1


For this:
Screen Shot 2020-05-18 at 11 12 27 AM
The CSS should be: 
```
background: linear-gradient(274.67deg, #FA7250 6.04%, #FF1893 44.31%, #A78AFF 100%);
border-radius: 0px 2px 2px 0px;
```

For this:
Screen Shot 2020-05-18 at 11 14 53 AM


The CSS should be:
```
/* Body Light Theme/13pt Poppins Regular 400 */

font-family: Poppins;
font-style: normal;
font-weight: normal;
font-size: 13px;
line-height: 24px;
letter-spacing: 0.01em;
```

The overall spacing of everything looks correct, but can you verify the sizing for the leftside navigation items and the spacing between that and the right content panel? It should look like this:

design QA 2

**Comment by petemill** [comment]
From @karenkliu :

> looking good! couple of minor issues:
> - dark theme icons in leftside nav should be dark theme/subtext grey400
> - unselected items in leftside nav should be Poppins Medium weight 500 (it's a lighter weight than the selected nav item, which you correctly have as Poppins Semibold 600)
> - nice work on the spacing, sizing, and gradients! :sparkles:

File: components/brave_new_tab_ui/containers/newTab/footerInfo.tsx [hunk 1]
```diff
@@ -80,30 +55,21 @@ export default class FooterInfo extends React.PureComponent<Props, {}> {
             </PhotoName>
           </S.GridItemCredits>
         }
+        {
+          <S.GridItemNavigationBraveToday>
+            ⬇🟢⬇🟢⬇🟢⬇🟢⬇🟢 scroll to see more ⬇🟢⬇🟢⬇🟢⬇🟢⬇🟢
```
[the remark this answers is not in the document]
Comment (petemill) [reply components/brave_new_tab_ui/containers/newTab/footerInfo.tsx]: I guess I had this as a pending comment from a while ago. No longer relevant AFAIK!

[the remark this answers is not in the document]
Comment (ryanml) [reply components/brave_new_tab_ui/containers/newTab/footerInfo.tsx]: yep was removed

**Comment by ryanml** [comment]
Jenkins failure unrelated, builds generated for each platform

**Pull request merged**